### PR TITLE
added tabindex to controls

### PR DESCRIPTION
**Pull request by chadzink**
This needs to be tested before merging.

### CODE REVIEW

**Comment by jweakley** [comment]
bump

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "ember-bootstrap-controls",
-  "version": "0.0.8",
+  "version": "0.0.9",
```
Comment (jweakley) [inline package.json]: We should handle version bumping after a PR has been accepted and merged to master.

File: addon/templates/components/bootstrap-selectize.hbs [hunk 2]
```diff
@@ -27,7 +27,8 @@
           update-filter=updateFilter
           select-item=selectItem
           add-item=addItem
-          remove-item=removeItem}}
+          remove-item=removeItem
+          tabindex=tabindex}}
```
Comment (jweakley) [inline addon/templates/components/bootstrap-selectize.hbs]: Doesn't look like this is supported: https://github.com/miguelcobain/ember-cli-selectize#properties or https://github.com/brianreavis/selectize.js/blob/master/docs/usage.md#options

File: addon/templates/components/bootstrap-selectize.hbs [hunk 3]
```diff
@@ -51,6 +52,7 @@
         update-filter=updateFilter
         select-item=selectItem
         add-item=addItem
-        remove-item=removeItem}}
+        remove-item=removeItem
+        tabindex=tabindex}}
```
Comment (jweakley) [inline addon/templates/components/bootstrap-selectize.hbs]: Same as above

**Comment by Samsinite** [comment]
Yeah, this doesn't look like it would set a tabindex. Since none of the projects currently need a tab index, we should just kill this PR until it is needed.

**Comment by jweakley** [comment]
Should we just approve this PR but not do a new version release?

**Comment by chadzink** [comment]
We could also just kill PR this because it does not really solve the issue.

**Pull request closed without merging**